contracts/interfaces/IStRSR.sol

```diff
@@ -83,19 +83,19 @@ interface IStRSR is IERC20MetadataUpgradeable, IERC20PermitUpgradeable, ICompone
     );
 
     /// Emitted whenever the exchange rate changes
-    event ExchangeRateSet(uint192 indexed oldVal, uint192 indexed newVal);
+    event ExchangeRateSet(uint192 oldVal, uint192 newVal);
 
     /// Emitted whenever RSR are paids out
-    event RewardsPaid(uint256 indexed rsrAmt);
+    event RewardsPaid(uint256 rsrAmt);
 
     /// Emitted if all the RSR in the staking pool is seized and all balances are reset to zero.
-    event AllBalancesReset(uint256 indexed newEra);
+    event AllBalancesReset(uint256 newEra);
```

Comment:
Let's leave indexed here since it runs so infrequently and it might be useful to query